<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmgment</class>
 <widget class="QWidget" name="fmgment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1331</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_hmjx">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hmgs">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免区域：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_xinjiang">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>新疆</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_huabei">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>华北</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免航班：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hmzlk">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_huadong">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>华东</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>设置时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_settime">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_starttime_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_dongbei">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>东北</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免机型：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hmjc">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hmhb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_xinan">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>西南</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免公司：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免走廊口：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_zhongnan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>中南</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_starttime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>豁免机场：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_endtime_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_xibei">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>西北</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_endtime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_end">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OK">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>631</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>21</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hmjcts">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>660</width>
     <height>460</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_plot"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>470</y>
     <width>660</width>
     <height>460</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_outComing"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
